frontend/pages/wiki/_page.vue

```diff
@@ -1,7 +1,5 @@
 <template>
   <v-container>
-    <search-simple />
-    <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Please enable eslint plugin in you local development and recover this line to avoid a eslint warning.